templates/common/members/functionary.html

```diff
@@ -11,7 +11,7 @@
             <div class="content">
                 <!-- Functionary Table Section -->
                 <h4>Funktionärsposter</h4>
-                <table class="table table-dark table-striped">
```

Comment:
KK isn't using functionary yet, also will this not break DaTe?

Reply:
Yes, on closer inspection this change would break DaTe I just forgot to check where i was making changes.